### PR TITLE
Add verification in samples

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:retest

File: gradle/apps/rest/src/main/java/com/liferay/blade/samples/rest/UsersRestService.java [hunk 1]
```diff
@@ -60,6 +61,13 @@ public String getUsers() {
 		return result.toString();
 	}
 
+	public void showLog() {
```
Comment (gamerson) [inline gradle/apps/rest/src/main/java/com/liferay/blade/samples/rest/UsersRestService.java]: No one calls show log.  We should just add the logging in the component rest method or in the activate so they can see it activating.

File: gradle/extensions/doclib-resource-command-override/src/main/java/com/liferay/blade/samples/doclib/resourcecommand/override/CustomMVCResourceCommand.java [hunk 2]
```diff
@@ -60,6 +61,11 @@ public boolean serveResource(
 		return true;
 	}
 
+	public void showLog() {
```
Comment (gamerson) [inline gradle/extensions/doclib-resource-command-override/src/main/java/com/liferay/blade/samples/doclib/resourcecommand/override/CustomMVCResourceCommand.java]: No method calls this, so we need to add the log to another method that is called and possibly in the activate.

File: gradle/extensions/resource-bundle/src/main/java/com/liferay/blade/samples/resourcebundle/CustomResourceBundle.java [hunk 3]
```diff
@@ -37,11 +39,18 @@
 		return _resourceBundle.getKeys();
 	}
 
+	public void showLog() {
```
Comment (gamerson) [inline gradle/extensions/resource-bundle/src/main/java/com/liferay/blade/samples/resourcebundle/CustomResourceBundle.java]: move to activate

**Review by gamerson — CHANGES_REQUESTED** [review]

**Comment by lawrence-lee** [comment]
@gamerson, I've fixed this with the changes you requested.

**Comment by lawrence-lee** [comment]
ci:retest using this pull to test DXP SP7

**Comment by lawrence-lee** [comment]
@gamerson ready to go

**Pull request merged**